.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Remove this commit, no relation to this branch. Do this in another pull request.